Creando la pagina Programs de MACILE

YaArias opened the pull request:
Aqui creamos la pagina para presentar los programas que ofrece MACILE.

### Issues

_¿Qué se está resolviendo?_
*La pagina programs en En y Es.*
### Todos

_¿Qué queda por hacer?_
*Revisar los contenidos subidos.*
### Áreas impactadas

_¿Qué páginas o componentes están afectados?_
*el programa yml*
### Pasos para reproducir

_¿Qué necesitamos hacer para probar?_

YaArias committed:
Creando la pagina Programs de MACILE

Aqui creamos la pagina para presentar los programas que ofrece MACILE.

misscs commented on content/en/programs.yml:
```diff
@@ -0,0 +1,11 @@
+    meta_social_image:
+
+    headline: Scholarship programs, savatinos and summer programs. :)
+    intro: Homepage
```
Muy bien :) Aunque a mi me enca 🍫 😉

misscs commented:
@YaArias @Clarissa0512 Falta el el archivo de `.html` y crear los paths en `AppConfig.js`

YaArias committed:
Agregando el Html y appConfig de la pagina de programs.

Aqui estamos agregando la pagina html y appConfig que se nos olvido en el commit anterior
de la pagina programs.

misscs commented on AppConfig.js:
```diff
@@ -11,6 +11,8 @@ export const config = {
     views: [
       {tpl: `home`, name: `home`},
       {tpl: `example`, name: `example`},
+      {tpl: `programs`, name: `example`},
```
Voy a arreglar estoy pero debe ser `programs` en la llave `name:` porque queremos la dirrecion `macile.org/programs` 😝

misscs committed:
Arreglar el nombre de la dirección 

Queremos la dirección `macile.org/programs`. Está bien en español

misscs reviewed (dismissed):
👍  good job! ready to merge
